migrate/starter/starter.go

```diff
@@ -61,6 +62,11 @@ func checkStartVersion(args []string) version {
 	if err != nil {
 		return internalV2
 	}
+	if fs.Lookup("version").Value.String() == "true" {
+		fmt.Println("etcd version:", etcdversion.Version, "(allow legacy mode)")
```

Comment:
The version number in this mode is always '2.0.0 (allow legacy mode)'. It exec 2.0 for 2.0 data dir, and exec 0.4 for 0.4 data dir.